cromshell

```diff
@@ -1040,47 +1040,54 @@ function cleanup()
   fi  
 
   # Get all states if we don't specify one:  
-  if [[ ${#st} -eq 0 ]] ; then   
+  if [[ ${#st} -eq 0 ]] ; then
+    echo getting all states
```

Comment:
Because you output all states as you iterate through them, you don't need to output this message here.